lib/trivial.nix

```diff
@@ -92,11 +92,16 @@ rec {
 
   ## nixpks version strings
 
+  release-info = import ../release-info.nix;
+
   # The current full nixpkgs version number.
   version = release + versionSuffix;
 
   # The current nixpkgs version number as string.
-  release = lib.strings.fileContents ../.version;
+  release = release-info.release;
+
+  # Note: the first letter is bumped on every release.  It's an animal.
```

Comment:
The note might be more relevant at the location the value is defined.

Reply:
fixed in 49a9011